### ISSUE TITLE
Include the sum of tstates and bytes in the Z80 Tstate calculator

### ISSUE THREAD

**Issue by rastersoft** [davidblackuk/davidblackuk.github.io#1]
Hello:
I found your Tstate calculator really useful, but I think it would be even better if it includes the sum of the Tstates and the bytes. The best way would be to put the accumulated value until each instruction. An example:

```
2 |  7 |   | 2 |  7 | ld a,0
1 |  4 |   | 3 | 11 | inc a
1 | 11 |   | 4 | 22 | rst #38
2 | 13 | 8 | 6 | 30 | djnz -5
2 |  7 |   | 8 | 37 | ld b, 5
```
This would allow to easily calculate how many bytes needs a piece of code just by sustracting the initial byte number to the final byte number, and the same for Tstates (for conditionals, I think it is better to add the number of Tstates needed when the condition is not met).

**Comment by PsychedelicPalimpsest** [comment]
I agree this is so useful although it would be nice to have a built-in t-state counter. I created a quick tampermonkey script to do this, if anybody else wants it, you can find it [here](https://pastebin.com/u0hVtBwk).

**Comment by hackerb9** [comment]
I also agree it would be useful, particularly if one could just select the lines to sum with the mouse and it showed the result as a tooltip. 

However, since that might be tricky, I have a simpler request. Please change the HTML output format so that it is easy to cut and paste into another tool. (For example, I can sum the second column of plain text using something like `awk '{sum+=$2} END {print sum}'`) As it is now, the HTML output looks fine visually, but the table gets mangled when pasted into a text file. I had to use my web browser's Inspector to copy the table as HTML and then run `pandoc` on it to massage it into a format that was easy to parse.

**Comment by PsychedelicPalimpsest** [comment]
The people have spoken @davidblackuk !
